tests/unit/BitbucketContext/Application/EventListener/PullRequestBuildStatusListenerTest.php

```diff
@@ -130,6 +130,22 @@ public function testItDoesNothingIfTheRepositoryDisabledTheInspections()
         $this->listener->onInspectionStarted($event);
     }
 
+    public function testItDoesNothingIfTheRepositoryIsInFlightMode()
+    {
+        $inspection = $this->createMock(Entity\PullRequestInspection::class);
+        $repository = $this->createMock(Entity\Repository::class);
+        $domainEvent = new Event\InspectionStarted($inspection);
+        $event = new DomainEventWrapper($domainEvent);
+
+        $inspection->method('getRepository')->willReturn($repository);
+        $repository->method('isInspectionEnabled')->willReturn(true);
+        $repository->method('isFlightModeEnabled')->willReturn(true);
+
+        $this->bitbucketClient->expects($this->never())->method('setBuildStatus');
```

Comment:
Call to an undefined method Regis\BitbucketContext\Application\Bitbucket\Client::expects().